scripts/analyze_csv_data.py

```diff
@@ -513,6 +512,8 @@ def improvements_plot(dfs, prefix, key, ascending):
             ticker.LogFormatterSciNotation(
                 labelOnlyBase=False,
                 minor_thresholds=(4, 2)))
+        ax.tick_params(axis='y', which='minor', labelsize=20)
```

Comment:
Two nits:
1. Both major and minor ticks can be changed with a single command by using which='both', instead of using tick_params and yticks as two separate commands.
2. In the infinite wisdom of matplotlib/pyplot, tick_params also exists as a pyplot method, meaning that the font change can be done by plt.tick_params(...) back at line 506 (where the old plt.yticks(...) command was).

Reply:
Ok I switched to that.